src/Http/Controllers/SwaggerViewController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace NextApps\SwaggerUi\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\ItemNotFoundException;
+
+class SwaggerViewController
+{
+    public function __invoke(Request $request)
+    {
+        try {
+            $file = collect(config('swagger-ui.files'))->filter(function ($values) use ($request) {
+                return ltrim($values['path'], '/') === $request->path();
+            })->firstOrFail();
+        } catch (ItemNotFoundException $e) {
+            return response()->json(['error' => 'File not found'], 404);
```

Comment:
```suggestion
        } catch (ItemNotFoundException) {
            return abort(404);
```
seems a bit weird to return json response on the view route